.gitmodules

```diff
@@ -0,0 +1,7 @@
+[submodule "libs/SpeedyKeyV"]
+	path = libs/SpeedyKeyV
+	url = https://github.com/ozxybox/SpeedyKeyV.git
+	branch = a
```

Comment:
Why a?

Reply:
According to Ozxy, they threw out the changes in a hurry. the branch: "a" the commit message: "a". Hey, it's not really my problem.

Reply:
What does this branch offer over the main branch though? I don't want to deal with changing it over if it gets merged or deleted.

Reply:
it has a bunch of functions (like the ability to create a KV object straight from a text file. rather than creating the object then calling parse), it also includes different methods compared to main, which are cleaner and less error prone. If you want a more permanent/proper branch naming, I can fork SpeedyKeyV and rename the branch. I'd rather have oxzy change the branch to either a more permanent name or merged with main. But yes, it is required over main.

Reply:
So, uh, what about this? In case you're also wondering, it's a required library by SAPP.

Reply:
I'm gonna mark this as Resolved, SpeedyKeyV is out of my control, if you like to see this changed ask Ozxy about SpeedyKeyV.